docs/UserGuide.adoc

```diff
@@ -47,6 +47,14 @@ Format: `help`
 Help is also shown if you enter an incorrect command e.g. `abcd`
 ====
 
+== Adding Junk : `junk`
+
+Format: `junk`
+
+[TIP]
```

Comment:
This seems like an introduction of your command, maybe it would be better to put it above the "Format: " line? Sort of like an explanation of your command, instead of it being a [TIP]. Also, try to follow the "standard" user guide format so that your teammates and you can produce a uniform writing style for a more user-friendly user guide. E.g. Instead of "A single line of junk is printed...", maybe rephrase to "Prints a single line of junk..." as you can see from the other commands' introductions "Adds a person.. ", "Deletes a person...".